src/ui/marker.js

```diff
@@ -604,4 +607,29 @@ export default class Marker extends Evented {
     getPitchAlignment() {
         return this._pitchAlignment;
     }
+
+    /**
+     * Sets the current `color` property of the marker.
+     * @param {string} [color] Sets the `color` property of the marker. If alignment is 'auto', it will automatically match `rotationAlignment`.
```

Comment:
similar to other setters this should describe the color paramater similar to https://github.com/mapbox/mapbox-gl-js/blob/36479a958e23d6f5d85bca914120956a5533eed7/src/ui/marker.js#L35. See setOffset offset for an example.

> If alignment is 'auto',... 

What that a mistake?

Reply:
Fixed.